Userland/Applications/Spreadsheet/Cell.h

```diff
@@ -121,17 +121,4 @@ struct Cell : public Weakable<Cell> {
     Format m_evaluated_formats;
 };
 
-class CellUndoCommand : public GUI::Command {
```

Comment:
isn't this class used in the code? looks like some rebase mess-up happened here, some changes in the next commit actually belong to this one: